### PR TITLE
Display client name in project details page

### PR DESCRIPTION
**Pull request by Kirannegi02**
Targets #1998

### Description
- The project details page displayed client name.
-  Resolved error message in input field (phone no).

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Client/Http/Requests/ClientFormsRequest.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ private function contactPersonsValidation()
         return [
             'client_contact_persons.*.name' => 'nullable|max:120',
             'client_contact_persons.*.email' => 'required|email',
-            'client_contact_persons.*.phone' => 'nullable|digits_between:4,10|integer',
+            'client_contact_persons.*.phone' => 'nullable|max:20',
```
Comment (P4NK4J) [inline Modules/Client/Http/Requests/ClientFormsRequest.php:30]: A phone number should ideally be 10 digits, I see here it is 20, let me know if I am missing something, or we need to change it to valid phone number

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left some comments, please check

File: Modules/Client/Http/Requests/ClientFormsRequest.php (hunk 1 above)
Comment (Kirannegi02) [reply Modules/Client/Http/Requests/ClientFormsRequest.php]: It was mentioned in the requirement that the phone no can accept input maximum upto 20 alphanumeric character and there are some clients which have code number such as (061) 707-2HG.

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Review by P4NK4J — APPROVED** [review]

**Pull request merged**